homework03/tasks.py

```diff
@@ -0,0 +1,42 @@
+
+# 1. Використовуючи функцію print, надрукувати фразу “I love Python” 42 рази.
+print("Task 1:")
+print("I love Python\n"*42)
```

Comment:
nit: better to use spaces before and after operations
```suggestion
print("I love Python\n" * 42)
```